tests/unit/test_preseq.yml

```diff
@@ -0,0 +1,32 @@
+- name: test_preseq_lc_extrap_no_defects
+  tags:
+    - unit
+    - unit_machine_2
+  command: >-
+    tests/caper_run.sh \
+    tests/unit/wdl/test_preseq_lc_extrap_no_defects.wdl \
```

Comment:
Should use same WDL, different JSON.

Reply:
oh boi, have I been autopiloting or what.